test/datapath_tests/poll_network_table_data.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
Thanks for setting up this file! will be useful to use for other dp tests as well.